packages/opentelemetry-plugin-fetch/src/fetch.ts

```diff
@@ -225,7 +228,9 @@ export class FetchPlugin extends core.BasePlugin<Promise<Response>> {
     response: FetchResponse
   ) {
     const endTime = core.hrTime();
-    spanData.observer.disconnect();
+    if (spanData.observer) {
```

Comment:
you can shorten the whole `if` to one liner as it was previously just add `?`
```suggestion
  spanData.observer?.disconnect();
```